src/outfitMakerProofOfConcept

```diff
@@ -0,0 +1,82 @@
+import java.util.ArrayList;
+
+import components.set.Set;
+import components.set.Set1L;
+
+public class OutMakerProofOfConcept {
+    private String shirts;
+    private String pants;
+    private String shoes;
+    private ArrayList<OutfitMakerProofOfConcept> outfit;
```

Comment:
This looks recursively defined. Maybe, you need an Outfit object, so you can store all three pieces.